Views/InputsView.swift

```diff
@@ -278,6 +290,7 @@ class InputsView: UIView, UITableViewDataSource, UITableViewDelegate, UISearchBa
     })
     isSearching = (searchText.isEmpty ? false : true)
```

Comment:
This is equivalent to `!searchText.isEmpty`, avoid useless conditionals if possible.